adaptive_ai/model_data/data/provider_models/groq_models_structured.yaml

```
@@ -1,14 +1,14 @@
 provider_info:
   name: GROQ
+provider_info:
```

YAMLlint (1.37.1)

[error] duplication of key "provider_info" in mapping

(key-duplicates)